src/Raven.Client/Documents/Commands/StartTransactionsRecordingCommand.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Net.Http;
+using Newtonsoft.Json;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Session;
+using Raven.Client.Http;
+using Raven.Client.Json;
+using Sparrow.Json;
+
+namespace Raven.Client.Documents.Commands
+{
+    public class StartTransactionsRecordingCommand : RavenCommand
+    {
+        private readonly string _filePath;
+
+        public StartTransactionsRecordingCommand(string filePath)
+        {
+            _filePath = filePath;
+        }
+        public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, ServerNode node, out string url)
+        {
+            url = $"{node.Url}/databases/{node.Database}/start_transactions_recording?file_path={_filePath}";
+
+            var request = new HttpRequestMessage
+            {
+                Method = HttpMethod.Get
+            };
+            return request;
+        }
+    }
+
+    public class StopTransactionsRecordingCommand : RavenCommand
+    {
+        public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, ServerNode node, out string url)
+        {
+            url = $"{node.Url}/databases/{node.Database}/stop_transactions_recording";
+
+            var request = new HttpRequestMessage
+            {
+                Method = HttpMethod.Get
+            };
+            return request;
+        }
+    }
+
+    public class ReplayTransactionsRecordingCommand : RavenCommand
```

Comment:
You shouldn't be providing a file path here, instead, you should upload the stream.
Note that here you _don't_ need to be an admin, just a normal user

Reply:
done